Traceback (most recent call last):
  File "/home/jahan/Missingness/train.py", line 139, in <module>
    metrics_df, perc_missing = run(
  File "/home/jahan/Missingness/benchmarkaux/openmlrun.py", line 76, in run
    X, y, classes, cat_bin = data.prepOpenML(dataset, task)
  File "/home/jahan/Missingness/UAT/datasets/data.py", line 500, in prepOpenML
    nan_mask = np.isnan(y)
  File "/home/jahan/.conda/envs/jax/lib/python3.10/site-packages/pandas/core/generic.py", line 2101, in __array_ufunc__
    return arraylike.array_ufunc(self, ufunc, method, *inputs, **kwargs)
  File "/home/jahan/.conda/envs/jax/lib/python3.10/site-packages/pandas/core/arraylike.py", line 397, in array_ufunc
    result = getattr(ufunc, method)(*inputs, **kwargs)
  File "/home/jahan/.conda/envs/jax/lib/python3.10/site-packages/pandas/core/arrays/categorical.py", line 1530, in __array_ufunc__
    raise TypeError(
TypeError: Object with dtype category cannot perform the numpy op isnan